src/app/assinante/component/assinante.component.ts

```diff
@@ -1,35 +1,78 @@
 import { Component, OnInit } from '@angular/core';
+import { response } from 'express';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/PadawansTeam/artmais-frontend/pullRequest?prid=8399665)
```suggestion
import { response } from "express";
```